.travis.yml

```diff
@@ -1,13 +1,14 @@
 ---
 language: node_js
 node_js:
-  - "0.12"
+  - "4"
```

Comment:
we should likely match ember-cli's here: https://github.com/ember-cli/ember-cli/blob/master/.travis.yml#L4-L8